chain-tx-enclave-next/mls/src/group.rs

```diff
@@ -561,7 +561,8 @@ pub struct GroupInfoPayload {
     /// that is processed)
     pub epoch: u64,
     /// 1..2^32-1
-    pub tree: Vec<Option<Node>>,
+    /// https://github.com/mlswg/mls-protocol/pull/345
+    pub tree: Vec<Node>,
```

Comment:
The PR may not be merged, so this can be reversed and put some FIXME note.
The final form is to be specified; it may end up something like `Vec<Option<NodeHash>>`

Reply:
Done.